test/sanity/code-smell/action-plugin-docs.py

```diff
@@ -1,39 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This file shouldn't be removed.

Reply:
Already add back, i.e. this file is not changed.